site/docs/tools/cli.md

```diff
@@ -25,19 +26,21 @@ Usage: nessie [OPTIONS] COMMAND [ARGS]...
   Interact with Nessie branches and tables via the command line
 
 Options:
-  --json           write output in json format.
-  -v, --verbose    Verbose output.
-  --endpoint TEXT  Optional endpoint, if different from config file.
+  --json             write output in json format.
+  -v, --verbose      Verbose output.
+  --endpoint TEXT    Optional endpoint, if different from config file.
+  --auth-token TEXT  Optional bearer auth token, if different from config
+                     file.
   --version
-  --help           Show this message and exit.
+  --help             Show this message and exit.
 
 Commands:
   branch       Branch operations.
   cherry-pick  Transplant HASHES onto current branch.
```

Comment:
"Current" is confusing here. It either merged into the default branch or the explicitly specified branch. Maybe just use "another"?

Reply:
`Transplant HASHES onto default branch or the explicitly specified branch` sounds pretty good and self-explaining

Reply:
```
  cherry-pick  Transplant HASHES onto default branch or the explicitly...
  config       Set and view config.
  contents     Contents operations.
  log          Show commit log.
  merge        Merge FROM_BRANCH into default branch or the explicitly...
```
I guess we'll use `another` then as the text is being shortened :(